include/meson.build

```diff
@@ -15,6 +15,7 @@ install_headers(
         'pms-utils/misc/macro-end.hpp',
         'pms-utils/misc/meta.hpp',
         'pms-utils/misc/x3_utils.hpp',
+        'pms-utils/misc/readfile.hpp'
```

Comment:
since this only contains internal functionality, perhaps move it to `internal` ? (I need to do the same for the meta headers too, admittedly)